src/api.ts

```diff
@@ -40,6 +40,24 @@ router.get('/spaces/:key', async (req, res) => {
   }
 });
 
+router.post('/deleted-spaces', async (req, res) => {
+  const { spaces } = req.body;
+  const id = (Array.isArray(spaces) ? spaces : [spaces])
+    .map(i => i?.toString())
+    .filter(a => a);
+
+  try {
+    return res.json(
+      id.length > 0
+        ? (await getDeletedSpaces(id)).map((s: { id: string }) => s.id)
+        : []
+    );
+  } catch (e: any) {
+    capture(e);
+    return sendError(res, 'server_error', 500);
+  }
+});
```

Comment:
Maybe we can make it a `get` call for now?

Reply:
GET call with a body ?

Reply:
No i mean, a endpoint that can return all deleted spaces, i feel like these spaces will be limited in number, right now there are 600 spaces, but looks like we want to check one space at a time so maybe we can use the route https://hub.snapshot.org/api/spaces/thanku.eth ? and return `deleted` property if spaces is deleted

Reply:
The endpoint accept a list spaces, so if user have a lot of ENS in his wallet, we can check all ENS in the same request